recipes/einsums/meta.yaml

```
@@ -0,0 +1,106 @@
+{% set name = "Einsums" %}
+{% set version = "0.2" %}
+{% set sha256 = "6f0e209add31b38d58a557e14afce7b23d5dd684a70aaf15554469f19ce77d29" %}
+{% set build_number = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build_number }}
+  string: {{ blas_impl }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
+  skip: true                                                               # [win]
+  binary_relocation: true
+  run_exports:
+    - {{ pin_subpackage('einsums', max_pin='x.x') }}
+  ignore_run_exports:
+    - elfutils                                                             # [linux]
+    - libzlib
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - llvm-openmp
```

As LLVM OpenMP is used with macOS builds and GNU OpenMP is used on Linux, would change these like so

```suggestion
    - llvm-openmp                                                          # [osx]
    - libgomp                                                              # [linux]
```